src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -18,12 +18,13 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
   <UsingTask TaskName="NuGet.Build.Tasks.Pack.PackTask" AssemblyFile="$(NugetTaskAssemblyFile)" />
   <UsingTask TaskName="NuGet.Build.Tasks.Pack.GetPackOutputItemsTask" AssemblyFile="$(NugetTaskAssemblyFile)" />
+  <UsingTask TaskName="NuGet.Build.Tasks.GetProjectTargetFrameworksTask" AssemblyFile="$(NugetTaskAssemblyFile)" />
```

Comment:
Consider using a property name that includes `Pack` in the name for the file path